CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - [BREAKING] Update to parse5 v4. For breaking changes see:
   - http://inikulin.github.io/parse5/#4-0-0 and
     http://inikulin.github.io/parse5/#3-0-0
+- [BREAKING] Split out the functions for walking the dom into `dom5.walking`.
+  So `dom5.nodeWalkAll` becomes `dom5.walking.nodeWalkAll`. These functions are
+  deprecated in favor of the faster and more flexible functions in
+  `dom5.iteration`. more easily track down and remove uses of them.
```

Comment:
"more easily track down and remove uses of them" <-- not sure what that part is saying